src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -9,29 +10,55 @@ import { ServiceContext } from '../../contexts/ServiceContext';
 import { isValidClick } from './clickTypes';
 
 const EVENT_TYPE = 'click';
+export const LITE_ATI_TRACKING = 'data-lite-ati-tracking';
+
+const useExtractTrackingProps = (props = {}) => {
```

Comment:
I realise that this is a hooks file, but this is not reallly a hook function. Can we rename it to `extractTrackingProps`, and just suppress the linting / typescript error? 

```suggestion
const extractTrackingProps = (props = {}) => {
```

Reply:
Is it not a hook? Maybe my definition of hooks is wrong, but this line I think makes it a hook: `const eventTrackingContext = useContext(EventTrackingContext);`